boilerplate/bin/setup

```diff
@@ -61,11 +61,14 @@ command -v react-native >/dev/null 2>&1 || {
 #   brew install applesimutils
 # }
 
-# IF NEEDED: Ensure detox-cli is installed
-# command -v detox >/dev/null 2>&1 || {
-#   echo "This app requires detox-cli to run end to end tests, but it was not found on your system."
-#   echo "Installing detox globally with npm..."
-#   npm install -g detox-cli
+# IF NEEDED: Ensure maestro is installed
+# command -v maestro >/dev/null 2>&1 || {
```

Comment:
We can add here installation for both ios and android. It's the same for both, just the last step is different for ios

Reply:
Yeah this would take care of both if the user wants it they can just uncomment. Should it be by default? Not sure because we didn't have the detox one uncommented either

Reply:
I think leaving this commented here is alright, since it matches the existing style